canonical-websites/www.ubuntu.com
# update /desktop/developers for vbt

## Done

* update /desktop/developers for vbt

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/desktop/developers)
- compare to the [design](https://www.dropbox.com/work/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/Patterns%20inventory/5.%20Desktop?preview=desktop-developers.png) and [demo](http://www.ubuntu.com-1656-desktop-developers.demo.haus/desktop/developers)

## Issue / Card

Fixes #1656
